### PR TITLE
fix: menu drawer was not visible

### PR DESCRIPTION
**Pull request by christian14b**
Menu drawer was not showing the components due to the way they were imported. With this change, I was able to load menu data when Lend button was active.

### CODE REVIEW

File: @kiva/kv-components/vue/KvWwwHeader/LendMenu/KvLendMenu.vue [hunk 1]
```diff
@@ -38,7 +38,7 @@ export default {
 	},
 	emits: ['load-lend-menu-data'],
 	setup(props, { emit }) {
-		const	userId = ref(null);
+		const	userId = ref(props.userId);
```
Comment (dyersituations) [inline @kiva/kv-components/vue/KvWwwHeader/LendMenu/KvLendMenu.vue:41]: Extra spaces after `const`

File: @kiva/kv-components/vue/KvWwwHeader.vue [hunk 2]
```diff
@@ -154,8 +153,9 @@ export default {
 		const searchOpen = ref(false);
 		const activeHeaderItem = ref(null);
 		const menuOpen = ref(false);
-		const menuComponent = ref(null);
```
Comment (dyersituations) [inline @kiva/kv-components/vue/KvWwwHeader.vue:157]: Why `shallowRef` here?

File: @kiva/kv-components/vue/KvWwwHeader/LendMenu/KvLendMenu.vue [hunk 3]
```diff
@@ -61,797 +61,47 @@ export default {
...
-			// 		}
-
-			// 	}`,
-			// }).subscribe({
-			// 	next: ({ data }) => {
-			// 		countryFacets.value = data?.data?.lend?.countryFacets ?? [];
-			// 		isRegionsLoading.value = false;
-			// 	},
-			// });
+			apollo.watchQuery({
```
Comment (dyersituations) [inline @kiva/kv-components/vue/KvWwwHeader/LendMenu/KvLendMenu.vue:64]: Do we need a loading state while this data is fetched? I know we had one in the old lend menu, so maybe it was already ported here, I just didn't see the changes in this PR.

**Review by dyersituations — APPROVED** [review]

File: @kiva/kv-components/vue/KvWwwHeader/KvHeaderLinkBar.vue [hunk 4]
```diff
@@ -150,9 +151,9 @@ import KvIconChevron from '../KvIconChevron.vue';
 import KvIconSearch from '../KvIconSearch.vue';
 import KvMaterialIcon from '../KvMaterialIcon.vue';
 
-const KvHeaderMobileMenu = () => import('./KvHeaderMobileMenu.vue');
-const KvHeaderMyKivaMenu = () => import('./KvHeaderMyKivaMenu.vue');
-const KvLendMenu = () => import('./LendMenu/KvLendMenu.vue');
+const KvHeaderMobileMenu = defineAsyncComponent(() => import('./KvHeaderMobileMenu.vue'));
+const KvHeaderMyKivaMenu = defineAsyncComponent(() => import('./KvHeaderMyKivaMenu.vue'));
+const KvLendMenu = defineAsyncComponent(() => import('./LendMenu/KvLendMenu.vue'));
```
Comment (emuvente) [inline @kiva/kv-components/vue/KvWwwHeader/KvHeaderLinkBar.vue:156]: Good catch!

File: @kiva/kv-components/vue/KvWwwHeader.vue (hunk 2 above)
Comment (emuvente) [reply @kiva/kv-components/vue/KvWwwHeader.vue:173]: +1 I can see shallowRef being useful for components references in general

File: @kiva/kv-components/vue/KvWwwHeader/LendMenu/KvLendMenu.vue (hunk 1 above)
Comment (emuvente) [reply @kiva/kv-components/vue/KvWwwHeader/LendMenu/KvLendMenu.vue:41]: +1

**Review by emuvente — APPROVED** [review]
Just needs that whitespace adjustment. Looks great!

**Pull request merged**